src/main/resources/db/migration/V4__create_sekvens_arbeidsfordeling_personopplysning.sql

```diff
@@ -0,0 +1,20 @@
+CREATE SEQUENCE arbeidsfordeling_pa_behandling_seq
```

Comment:
nitpick, men filnavnet inkluderer ikke oppgave som det også lages sekvens for

Reply:
endret det